workshops/templates/workshops/time_series.html

```diff
@@ -17,7 +17,7 @@
     <script src="{% static 'metrics-graphics/dist/metricsgraphics.js' %}"></script>
 
     <script type="text/javascript">
-        d3.json('{{ api_endpoint }}?format=json', function(data) {
+        d3.json('{{ api_endpoint }}&format=json', function(data) {
```

Comment:
Does this imply that the `api_endpoint` variable now includes the start of the query parameter string?